Tests/OctopusDSC.Tests.ps1

```diff
@@ -1,239 +1,268 @@
-$script:scriptRoot = $PSScriptRoot
-
-Describe "PSScriptAnalyzer" {
-    BeforeAll {
+Describe "OctopusDSC" {
+    BeforeDiscovery {
```

Comment:
Ooh - never head of this one

Reply:
Looks new in v5 https://pester-docs.netlify.app/docs/usage/discovery-and-run
TL;DR the test cases are generated in the discovery phase but you can't declare variables and such in this phase as they're discarded